docs/resources/password_expiration_policy.md

```diff
@@ -0,0 +1,42 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "artifactory_password_expiration_policy Resource - terraform-provider-artifactory"
+subcategory: "Security"
+description: |-
+  Provides an Artifactory Password Expiration Policy resource.
+---
+
+# artifactory_password_expiration_policy (Resource)
+
+Provides an Artifactory Password Expiration Policy resource.
+
+## Example Usage
+
+```terraform
+resource "artifactory_password_expiration_policy" "my-password-expiration-policy" {
+  name = "my-password-expiration-policy"
+  enabled = true
+  password_max_age = 120
+  notify_by_email = true
+}
+```
+
+## Argument reference
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `enabled` (Boolean) Enable Password Expiration Policy. This only applies to internal user passwords.
+- `name` (String) Name of the resource. Only used for importing.
+- `notify_by_email` (Boolean) Send mail notification before password expiration. Users will receive an email notification X days before password will expire. Mail server must be enabled and configured correctly.
```

Comment:
Do we know how many days? Where it's configured?

Reply:
Yes, I've just added the `artifactory_user_lock_policy` resource. I was originally going to open a separate PR but decided to fold that into this PR.

Reply:
Actually I don't think that 'X' days is configurable in the UI so may be it's in the configure yaml?